.github/workflows/build-test.yml

```diff
@@ -48,6 +48,7 @@ jobs:
               env:
                 PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
+                GORACE: history_size=7
```

Comment:
how is this used?

Reply:
When race occurs, only one stack was gathered (with latest access). In that way, we have access also to previous threads accessing the same space where race occurs.